src/main/java/picard/analysis/CompareMetrics.java

```diff
@@ -267,7 +267,11 @@ protected SimpleResult compareMetricValues(final Object value1, final Object val
             if (value1 instanceof Number) {
                 double numValue1 = ((Number) value1).doubleValue();
                 double numValue2 = ((Number) value2).doubleValue();
-                double relativeChange = (numValue1 == 0 && numValue2 == 0) ? 0 : (numValue1 == 0) ? Double.MAX_VALUE : (numValue1 - numValue2) / numValue2;
+                double relativeChange = 0;
+                if ((!Double.isNaN(numValue1)) || (!Double.isNaN(numValue2))) {
```

Comment:
Can use fewer parens
```suggestion
                if (!Double.isNaN(numValue1) || !Double.isNaN(numValue2)) {
```